src/frontend/Sider/Behandling/Vilkårvurdering/LesevisningVilkår.tsx

```diff
@@ -33,16 +28,12 @@ const Container = styled(FlexColumn)`
     box-shadow: ${AShadowXsmall};
 `;
 
-const FlexMedMargin = styled(FlexColumn)`
-    margin: 0 45px;
-`;
-
-const Svar = styled(Lesefelt)`
-    grid-column: 1;
-`;
-
-const Begrunnelse = styled(Lesefelt)`
-    grid-column: 2;
+const StyledButton = styled(SmallButton)`
```

Comment:
Hvis man kaller det for eks Redigeringsknapp så er det litt tydligere hva den brukes til
```suggestion
const Redigeringsknapp = styled(SmallButton)`
```